modules/ROOT/partials/configuration/sidebar_show.adoc

```diff
@@ -0,0 +1,27 @@
+[[sidebar_show]]
+== `+sidebar_show+`
+
+NOTE: This option is only available for TinyMCE 6.1 and later.
+
+This option allows you to show the specified sidebar on initialization.
+
+*Type:* `+String+`
+
+=== Example: Using `+sidebar_show+`
+
+[source,js]
+
+```javascript
+  tinymce.init({
+    setup: (editor) => {
+      editor.ui.registry.addSidebar('mysidebar', {
+        tooltip: 'My sidebar',
+        icon: 'comment',
+        onShow: (api) => {
+          api.element().innerHTML = 'Hello world!';
+        },
+      });
+    },
+    sidebar_show: 'mysidebar'
+  });
+```
```

Comment:
This isn't how you do an code example in AsciiDoc... it also doesn't follow the style guide: https://github.com/tinymce/tinymce-docs/blob/staging/docs-6/CONTRIBUTING.md#code-snippets